creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -465,7 +465,10 @@ const _additionalSyncIsRequired = async (
        * Note - secondaryClockValue can be greater than primaryClockValue if additional
        *    data was written to primary after primaryClockValue was computed
        */
-      if (secondaryClockValue >= primaryClockValue) {
+      if (
+        secondaryClockValue >= primaryClockValue &&
+        syncMode !== SYNC_MODES.MergePrimaryAndSecondary
```

Comment:
did you mean MergeAndWipe here?

Reply:
yep 🤦‍♂️ at least our tests were comprehensive enough to catch this as well, thanks for flagging

Reply:
hahaha, love that